src/test/ui/type-alias-impl-trait/impl_trait_for_same_tait.stderr

```diff
@@ -0,0 +1,30 @@
+error[E0119]: conflicting implementations of trait `Bop` for type `Bar<()>`
+  --> $DIR/impl_trait_for_same_tait.rs:18:1
+   |
+LL | impl Bop for Bar<()> {}
+   | -------------------- first implementation here
+...
+LL | impl Bop for Bar<i32> {
+   | ^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `Bar<()>`
+
+error[E0119]: conflicting implementations of trait `Bop` for type `Bar<()>`
+  --> $DIR/impl_trait_for_same_tait.rs:27:1
+   |
+LL | impl Bop for Bar<()> {}
+   | -------------------- first implementation here
+...
+LL | impl Bop for Barr {
+   | ^^^^^^^^^^^^^^^^^ conflicting implementation for `Bar<()>`
```

Comment:
I'm not sure yet how to improve this diagnostic beyond looking at the entries in `take_opaque_types()` and doing some guessing for common cases.